python/cudf/cudf/dataframe/columnops.py

```diff
@@ -230,18 +231,31 @@ def build_column(buffer, dtype, mask=None, categories=None, name=None):
             ordered=False,
             mask=mask,
             name=name,
+            null_count=null_count
         )
```

Comment:
Move this to the bottom instead of the top?

Reply:
@kkraus14 , done. Actually made that change else where in codebase too.